python/lsst/ts/IntegrationTests/aux_tel_track_target.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of ts_IntegrationTests
```

Comment:
This is not what I had in mind. Each track_target script needs to be followed by the visit image sequence scripts. I think we need to talk about this one more if you don't understand what I'm after here.

Reply:
Fixed.  I think.